src/compton/test/tCompton.cc

```diff
@@ -98,6 +103,17 @@ void compton_file_test(rtt_dsxx::UnitTest &ut) {
   if (!soft_equiv(interp_data[0][0][3], 3.59663442e-02))
     ITFAILS;
 
+  if (!soft_equiv(interp_nu_data[0][0], 1.5000000e+00))
```

Comment:
If you need to make this check tighter, soft_equiv takes a 3rd argument that is the relative tolerance for the check.